tests/usecases/data-processing-cloudant/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: cloudant_instance
+  version: 1.0
+  license: Apache-2.0
+  binding-package: /whisk.system/cloudant
+  inputs:
+    username: cloudant_user
+    password: cloudant_password
+    host: cloudant_user.cloudant.com
+  actions:
+    write-to-action:
+      version: 1.0
+      location: actions/write-to-cloudant.js
+      runtime: nodejs
+    write-from-action:
+      version: 1.0
+      location: actions/write-from-cloudant.js
+      runtime: nodejs
+  triggers:
+    image-uploaded:
+      feed: cloudant_instance/changes # cloudant_instance is the same as the package binding name
+      inputs:
```

Comment:
we're now overloading inputs to do in-line parameter binding so setting schema information here doesn't have any effect.